Task1/Project/Shapes/Models/Classes/Circle.cs

```diff
@@ -74,16 +68,30 @@ public Point Center
             }
         }
         // METHODS
-        public override void ReadFromFile(StreamReader readStream)
+        /// <summary>
+        /// Reads some information about circle from file.
+        /// </summary>
+        /// <param name="readStream">
+        /// Stream for easy consistent access to file and is only for reading from it.
+        /// </param>        
+        public override void ReadFromFile(System.IO.StreamReader readStream)
         {
-            throw new NotImplementedException();
+            string[] data = readStream.ReadLine().Split(' ');
+            center.X = double.Parse(data[0]);
+            center.Y = double.Parse(data[1]);
+            radius = double.Parse(data[2]);        

```

Comment:
Should also check amount of parameters and throw exception if it needed

Reply:
Ok.